docs/tutorials/05-model-monitoring.ipynb

```diff
@@ -11,14 +11,16 @@
     "(realtime-monitor-drift-tutor)=\n",
```

Comment:
<p>i think it should be remove and maybe add a note that if the user want to use S3 storage the use can see also this link -<a href="https://docs.mlrun.org/en/stable/store/datastore.html#s3" target="_blank">https://docs.mlrun.org/en/stable/store/datastore.html#s3</a>.</p><p>in addition, using S3 and how to use it out of this tutorials scope and may confuse users </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/8410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>